conf/config.yaml

```diff
@@ -98,7 +98,10 @@ nginx_config:                     # config for render the template to genarate n
       - 'unix:'
 
 etcd:
-  host: "http://127.0.0.1:2379"   # etcd address
+  host:
+    - "http://127.0.0.1:2379"     # multiple etcd address
```

Comment:
That is wrong. By default, there should be only one etcd address.

Reply:
oh sure and multiple http_hosts. I got it wrong

Reply:
Sure fixed